src/lib/json/json.c

```diff
@@ -45,6 +45,9 @@
 int fr_json_object_to_value_box(TALLOC_CTX *ctx, fr_value_box_t *out, json_object *object,
 				fr_dict_attr_t const *enumv, bool tainted)
 {
+	rad_assert(out != NULL);
+	rad_assert(object != NULL);
+
```

Comment:
The preferred way is to use CC_HINT(nonnull) in the header file.  Look for existing examples ``CC_HINT(nonnull(2,3))`` would be the markup for this case